_posts/2022-03-16-launch-announcement-1-3-0.markdown

```diff
@@ -0,0 +1,109 @@
+
+* **App Analytics**: In the past, users had to collate logs, traces, and metrics in separate views which made application monitoring difficult. With App Analytics Dashboards, users can now view application logs, traces, and metrics in one view instead of moving between different visualizations.
+
+![App Analytics Dashboard]({{ site.baseurl }}/assets/media/blog-images/2022-03-16-Launch-Announcement-1-3-0/App-Analytics.gif){: .img-fluid }
+
+* **Trace ID Correlation**: Users who regularly track events across applications will now be able to use correlation ID (based on the [Open Telemetry specification](https://opentelemetry.io/docs/reference/specification/logs/overview/#log-correlation)) to tie events together when viewing events and in-context visualizations.
+
+![Trace log Analytics]({{ site.baseurl }}/assets/media/blog-images/2022-03-16-Launch-Announcement-1-3-0/trace-log-correlation.gif){: .img-fluid }
+
+* **Live Tail**: Previously, when users were watching a live event take place, they had to manually refresh their view. Users can now configure the interval in which content is refreshed saving the hassle of manually refreshing.
+
+![Live Tail View]({{ site.baseurl }}/assets/media/blog-images/2022-03-16-Launch-Announcement-1-3-0/Live-Tail.gif){: .img-fluid }
+
+* **Field Insights and Enhanced Visualization Support**: Visualization types such as Pie and heat maps were added as well as the ability to add named threshold markers as lines on the visualizations. When users want to know more about a metric that they are looking at, they can select view surrounding events to get a correlated picture. In the event a user needs a reference point for a future meeting, visualizations can now be saved to notebooks for convenience.
+
+![Field Insights View]({{ site.baseurl }}/assets/media/blog-images/2022-03-16-Launch-Announcement-1-3-0/field-insights.gif){: .img-fluid }
+
+### SQL and PPL
+
+* **PPL Runtime Fields**: Users may opt out of formatting the schema upon writing the index (schema-on-write). This opt out can be used to speed up indexing time, or for additional presentation flexibility. Users who want to define their schema upon querying the index (schema-on-read), can now use regex and parse commands to further format their schema using PPL.
+
+![PPL runtime workflow]({{ site.baseurl }}/assets/media/blog-images/2022-03-16-Launch-Announcement-1-3-0/PPL-Parse.gif){: .img-fluid }
+
+
+* **Machine Learning Support in PPL (Piped Processing Language):** Users can now process arbitrary observability events using Anomaly Detection based on Random Cut Forest (RCF) and KMeans commands in PPL.
+* **Improved Aggregate Functions Support in SQL and PPL**: Users expect to use aggregate functions in SQL and PPL. OpenSearch SQL and PPL now supports ORDER BY and default query limit size.
```

Comment:
ORDER BY should be in backticks (``)

Reply:
Can you show me an example of where it is common to use a backtick for this scenario?

Reply:
Got it!